dart/shelf/pubspec.yaml

```
@@ -0,0 +1,8 @@
+name: shelf_app
+
+environment:
+  sdk: ^3.0.0
+
+dependencies:
+  shelf: 1.4.1
+  shelf_router: 1.1.4
```

maybe
```suggestion
  shelf_router: *
```
not sure there is a way to define latest stable in dart (like `@stable` in php)